src/screens/List/ListHiddenScreen.tsx

```diff
@@ -0,0 +1,203 @@
+      } catch (e) {
+        setIsProcessing(false)
+        logger.error('Failed to unblock list', {message: e})
+        Toast.show(
+          _(
+            msg`There was an issue. Please check your internet connection and try again.`,
+          ),
+        )
+        return
+      }
+    }
+    Toast.show(_(msg`Unsubscribed from list`))
+    setIsProcessing(false)
+  }
+
+  const onRemoveList = async () => {
+    if (!savedFeedConfig) return
+    try {
+      await removeSavedFeed(savedFeedConfig)
+      Toast.show(_(msg`Removed from saved feeds`))
+    } catch (e) {
+      logger.error('Failed to remove list from saved feeds', {message: e})
+      Toast.show(
+        _(
+          msg`There was an issue. Please check your internet connection and try again.`,
+        ),
+      )
+    } finally {
+      setIsProcessing(false)
+    }
+  }
+
+  return (
+    <CenteredView
+      style={[
+        a.flex_1,
+        a.align_center,
+        a.gap_5xl,
+        !gtMobile && a.justify_between,
+        t.atoms.border_contrast_low,
+        {paddingTop: 175, paddingBottom: 110},
+      ]}
+      sideBorders={true}>
+      <View style={[a.w_full, a.align_center, a.gap_lg]}>
+        <EyeSlash
+          style={{color: t.atoms.text_contrast_medium.color}}
+          height={42}
+          width={42}
+        />
+        <View style={[a.gap_sm, a.align_center]}>
+          <Text style={[a.font_bold, a.text_3xl]}>
+            <Trans>List has been hidden</Trans>
+          </Text>
+          <Text
+            style={[
+              a.text_md,
+              a.text_center,
+              a.px_md,
+              t.atoms.text_contrast_high,
+              {lineHeight: 1.4},
+            ]}>
+            <Trans>
+              This list - created by{' '}
+              <Text style={[a.text_md, !isOwner && a.font_bold]}>
+                {isOwner
+                  ? _(msg`you`)
+                  : sanitizeHandle(list.creator.handle, '@')}
+              </Text>{' '}
+              - contains possible violations of Bluesky's community guidelines
+              in its name or description.
+            </Trans>
+          </Text>
+        </View>
+      </View>
+      <View style={[a.gap_md, gtMobile ? {width: 350} : [a.w_full, a.px_lg]]}>
+        <View style={[a.gap_md]}>
+          {savedFeedConfig ? (
+            <Button
+              variant="solid"
+              color="secondary"
+              size="medium"
+              label={_(msg`Remove from saved feeds`)}
+              onPress={onRemoveList}
+              disabled={isProcessing}>
+              <ButtonText>
+                <Trans>Removed from saved feeds</Trans>
+              </ButtonText>
+            </Button>
+          ) : null}
+          {isOwner ? (
+            <Button
+              variant="solid"
+              color="secondary"
+              size="medium"
+              label={_(msg`Show list anyway`)}
+              onPress={() => setIsContentVisible(true)}
```

Comment:
This method won't do anything if `noOverride` is set. Kinda just future-proofing, but might be good to conditionally render this button. Alternatively, we remove that guard and force ourselves to check `meta.noOverride` on the hider context.